GenderPayGap.WebUI/Models/Shared/ReportStatusBadgeViewModel.cs

```diff
@@ -1,13 +1,15 @@
 ﻿using GenderPayGap.Core;
 
-namespace GenderPayGap.WebUI.Models
+namespace GenderPayGap.WebUI.Models.Shared
 {
 
     public class ReportStatusBadgeViewModel
     {
         public ReportStatusBadgeType ReportStatus { get; set; }
         public string DateText { get; set; }
         public bool Desktop { get; set; }
+        public string DesktopClasses { get; set; }
+        public string MobileClasses  { get; set; }

```

Comment:
If we're already telling it whether it's on desktop or not, why do we need to differentiate between desktop and mobile classes? Are we ever going to need to provide both?

Reply:
Yeah, good spot, I've adjusted the this.